relay-server/src/services/processor.rs

```diff
@@ -503,6 +506,7 @@ impl ProcessingError {
             // These outcomes are emitted at the source.
             Self::MissingProjectId => None,
             Self::EventFiltered(_) => None,
+            Self::ReplayFiltered(_) => None,
```

Comment:
I think this should map to `Outcome::Filtered(FilterStatKey)`. Then `ManagedEnvelope::reject` will automatically emit outcomes for the envelope's `replay_quantity` (can be verified in the integration test).

Reply:
The integration tests are tricky, sometimes the default consumer timeout is not enough for relay to produce all the data. I verified locally that you get an outcome in the integration test with `outcomes_consumer.get_outcome(timeout=5)`.

Reply:
My mistake.  I wasn't aware.  Thank you!!